internal/registry/registry.go

```diff
@@ -173,14 +176,14 @@ func pkgInfoFromPath(srcDir string, mode packages.LoadMode) (*packages.Package,
 	return pkgs[0], nil
 }
 
-func findPkgPath(pkgInputVal string, srcPkg *packages.Package) string {
+func findPkgPath(pkgInputVal string, srcPkgPath string) string {
```

Comment:
I changed the argument so that we don't pass more into the function than what's needed.